src/Reducers/UserFormStateReducer.js

```diff
@@ -2,21 +2,13 @@ import { produce, enableMapSet } from "immer";
 
 enableMapSet();
 
-export const EDIT_NAME = "EDIT_NAME";
-export const EDIT_PREFERRED_NAME = "EDIT_PREFERRED_NAME";
 export const EDIT_EMAIL = "EDIT_EMAIL";
 export const EDIT_ROLE = "EDIT_ROLE";
 export const EDIT_PROGRAMS = "EDIT_PROGRAMS";
```

Comment:
This technically isn't needed for now, but I can leave it here for when organization user access gets built.